### PR TITLE
Bug 	1325329 - Allow improper creation dates

### PR DESCRIPTION
**Pull request by fbertsch**
For 2017-01-09, .03% of the release data had malformed creation
dates. Since it's such a small number, we are just going to ignore
those.

I tested this on 100% of the release data for 2017-01-09.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla-services/data-pipeline/239)

### CODE REVIEW

File: reports/mobile_clients/mobile-clients.ipynb [hunk 1]
```diff
@@ -73,11 +73,15 @@
...
-    "        # This is only accurate because we know the creation date is always in 'Z' (zulu) time.\n",
-    "        creationDate = dt.datetime.strptime(ping[\"created\"], \"%Y-%m-%d\")\n",
+    "    # Create date can be an improper string (~.03% of the time, so ignore)\n",
+    "    try: \n",
+    "        # Create date should already be in ISO format\n",
+    "        creationDate = ping[\"created\"]\n",
+    "        if creationDate is not None:\n",
+    "            # This is only accurate because we know the creation date is always in 'Z' (zulu) time.\n",
+    "            creationDate = dt.datetime.strptime(ping[\"created\"], \"%Y-%m-%d\")\n",
+    "    except Exception:\n",
```
Comment (maurodoglio) [inline reports/mobile_clients/mobile-clients.ipynb]: It would be better to specify the exact exception you want to catch. It should be ValueError iirc.

**Review by maurodoglio — COMMENTED** [review]
Just a minor change and it's ready to go

**Comment by vitillo** [comment]
What about moving this notebook to RTMO so we don't have to review JSON files? :)

**Comment by fbertsch** [comment]
Sorry, I can put in the exact error but I want to go back and test it on the data to make sure it's catching the right thing. Spark reported an UnicodeEncodeError.

**Comment by fbertsch** [comment]
@vitillo what if I LIKE reviewing JSON files? Hmmmmm??

**Comment by vitillo** [comment]
Ohhh, you are one of those people... ;-D

**Comment by fbertsch** [comment]
@maurodoglio, I tested out with ValueError and it indeed catches all exceptions. RFAL.

**Pull request merged**